### DISCUSSION TITLE
Any headless markdown CMS good for FixIt template?

### DISCUSSION THREAD

**Discussion by crimcol** [hugo-fixit/FixIt#249] [General]
Hello,
I'm looking for cms which would work nice with this template.
What I've tried already is NetlifyCMS and Forestry.
They both can't save images in the same folder with .md file.
NetlifyCMS does not support subfolders at all... or I didn't figure out.

What CMS do you use?

**Comment by Lruihao** [comment]
I am using Github page or Vercel.
